templates/cassandra-22.conf.jinja

```diff
@@ -602,4 +602,152 @@ limitations under the License.
                 Attribute "Value"
             </Value>
         {%- endcall %}
+        {% call mbean("cassandra_Cache_KeyCache-HitRate", mbeans) -%}
+            ObjectName "org.apache.cassandra.metrics:type=Cache,scope=KeyCache,name=HitRate"
+            <Value>
+                Type "gauge"
+                InstancePrefix "cache_key_cache-hit_rate"
+                Table false
+                Attribute "Value"
+            </Value>
+        {%- endcall %}
+        {% call mbean("cassandra_ClientRequest_Read-Latency", mbeans) -%}
+            ObjectName "org.apache.cassandra.metrics:type=ClientRequest,scope=Read,name=Latency"
+            InstancePrefix "client_request_read-latency"
```

Comment:
Really sorry to make you undo this, but it took me a while to swap the context back in and remember how we were mapping Java metrics. We have other instances of this mapping in, e.g., [hbase-098.conf.jinja](https://github.com/Stackdriver/stackdriver-agent-service-configs/blob/master/templates/hbase-098.conf.jinja#L79) and [jvm-sun-hotspot-macros.jinja](https://github.com/Stackdriver/stackdriver-agent-service-configs/blob/master/templates/jvm-sun-hotspot-macros.jinja#L26), and all of them use the TypeInstance exclusively (not PluginInstance). And I now remember the reason for this -- we wanted to allow users to distinguish between multiple instances of the application running on the same VM, so PluginInstance was reserved for that. The GC metrics are actually an aberration, not the norm.

Reply:
It's not a problem. Reverted last commit.